src/SignalR/common/testassets/Tests.Utils/VerifyNoErrorsScope.cs

```diff
@@ -34,6 +34,33 @@ public void Dispose()
 
             var results = _sink.GetLogs().Where(w => w.Write.LogLevel >= LogLevel.Error).ToList();
 
+            #region RemoveIn3_0
```

Comment:
Could we use `#if NETCOREAPP2_2` or similar to make extra sure this doesn't run in 3.0 even if we mess up and accidentally merge the code?